stylesheet.css

```diff
@@ -2,3 +2,109 @@ body {
   background-color: white;
   font-family: 'Lato', sans-serif;
 }
+
+fieldset {
+  background-color: white;
+  width: 40vw;
+}
```

Comment:
- [ Optional ] Kindly adjust the alignment of this filed-set just like the [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). 
![Capture](https://user-images.githubusercontent.com/93598189/226968466-e949f66b-d58b-44b2-a746-551d18337e3f.PNG)